content/ui-patterns/feature-onboarding.mdx

```diff
@@ -0,0 +1,271 @@
+
+## Dashboard feature bulletin
+The top of the right sidebar in a user’s dashabord (logged-in home) is a designated area for feature discovery. They take the user directly to the feature in the product or the relevant marketing pages.
+
+**Basic announcement guidelines**
+• Limit messages to around 160 characters. 
+• If there is a click through, include a clear call-to-action button; avoid hidden links.
+• Use color, with discretion, to spotlight and bring brand feature personality to the annoucement.
+
+<Flex alignItems="center" flexDirection="column" my={5}>
+  <BorderBox width={12 / 12} border="1px solid" borderColor="gray.3" mb="2">
+    <img src="https://user-images.githubusercontent.com/66705495/88829133-09957e80-d181-11ea-9ab2-81a6c5513e20.png" />
+  </BorderBox>
+  <p>GitHub Actions branded first time user experience</p>
+</Flex>
+
+## Beta feature preview dialog
+Feature previews allow teams to test features in production with users who opt in. A feature under the beta preview will be added to the Feature Preview dialog, accessible through the user dropdown
+
+Please note, features in Beta testing are strongly advised to have a comprehensive release and redaction plan. No feature should remain in Beta for longer than X months unmonitored.
+
+<Flex alignItems="center" flexDirection="column" my={5}>
+  <BorderBox width={12 / 12} border="1px solid" borderColor="gray.3" mb="2">
+    <img src="https://user-images.githubusercontent.com/66705495/88829215-2762e380-d181-11ea-956e-8cf4166ac92c.png" />
+  </BorderBox>
+  <p>GitHub Actions branded first time user experience</p>
+</Flex>
+
+
+# Flairs
+Flairs are subtler ways to accentuate new features. They are a good alternative to call attention to new things without derailing the user from their primary job to be done. 
+
+Consider using flairs for features that appear in a user’s current view but may navigate away from their current page. They may also be applicable for smaller features that appear inline that more self-explainatory and do not need a teaching bubble. 
+
+See Primer documentation on labels and muted links for detailed implementation guidelines.
+
+**Beta**
+Beta tags are applicable for feature previews. They may be accompanied by a feedback link which is encouraged but not required, depending on space constraints. 
+<Flex alignItems="center" flexDirection="column" my={5}>
+  <BorderBox width={12 / 12} border="1px solid" borderColor="gray.3" mb="2">
+    <img src="https://user-images.githubusercontent.com/66705495/88829463-7872d780-d181-11ea-849c-5c293d40ccea.png" />
+  </BorderBox>
+  <p>GitHub Actions branded first time user experience</p>
+</Flex>
+<Flex alignItems="center" flexDirection="column" my={5}>
+  <BorderBox width={12 / 12} border="1px solid" borderColor="gray.3" mb="2">
+    <img src="https://user-images.githubusercontent.com/66705495/88829510-8c1e3e00-d181-11ea-9218-bdca1c08d819.png" />
+  </BorderBox>
+  <p>GitHub Actions branded first time user experience</p>
```

Comment:
There's a lot of captions with this same sentence, can we make them more clear, like "An example of first-time user experiences: GitHub Actions" -- there's probably a nicer way of phrasing it, this is just a first idea 😄